addons/website_slides/controllers/main.py

```diff
@@ -754,6 +769,17 @@ def slide_quiz_submit(self, slide_id, answer_ids):
             'quizAttemptsCount': quiz_info['quiz_attempts_count'],
             'rankProgress': rank_progress,
         }
+    
+    @http.route(['/slides/slide/quiz/save_slide_answsers'], type='json', auth='public', website=True)
```

Comment:
`answsers` -> `answers`

Please grep it to change everywhere.

I noticed you have a few spelling mistakes in your code and in your commit message.
I personally use a spell checker in my VS Code extension ("Code Spell Checker") and it helps for that kind of things.
It shows up like that for e.g:
![Selection_309](https://user-images.githubusercontent.com/41282536/67264627-f2b45400-f4ab-11e9-8b13-623e377a7fb5.png)